invenio_records_marc21/ui/records/wrappers.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# Invenio-Records-Marc21 is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Records string wrapper."""
+
+from flask_babelex import gettext as _
+
+PERSONAL_CODES = {
```

Comment:
would it be possible to move that into a vocabulary? i would prefer not to hard code that dict

Reply:
Yes, but needs further investigation.

Reply:
ok, then we keep it that way for now and we open an issue to handle that later on